_posts/2015-04-28-feedhenry-push-templates.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: news
+section: news
+title: AeroGear quick start push Templates
+author: edewit
+---
+
+Currently in software development the biggest challenge is integrating. A lot of what we do involves integrating with other platforms or libraries. Knowing how to integrate is important, so here I'm going to show how to integrate the Unified Push Server (UPS) with Feedhenry, for adding push notification support. To make things even easier we created a Feedhenry Template that you can use as a starting point for your application that contains all the parts you'll need.
+
+To illustrate how to use the FeedHenry Template for push notifications, you will build a set of sample applications for receiving sports news. You will use a centralized cloud application that integrates with both a push server as well as any client applications. The centralized cloud app contains various categories (i.e. different sports) and news stories associated with one or more categories. There are also two client applications: 
+
+* Console web application: Let administrators create and update the category list, as well as create news stories on the centralized cloud app
+* Mobile application: Lets users subscribe to any of the available categories and receive a push notification whenever a new story is posted with that category
+
+To build a project based on this template:
+
+1. Create a new project.
+1. Click *Sample Projects*, and then find the Project named: *Push Hello World*.
+1. Click select type a name, and then click *create*.
+
+![choose template](/img/news/2015-04-20-feedhenry-push-templates/1.png)
+
+Now you have three apps one Cloud Code App and two Client Apps.
+
+![app structure](/img/news/2015-04-20-feedhenry-push-templates/5.png)
+
+The _Push Cloud App_ is a simple server-side CRUD application for categories. The _Push Console App_ is a "management inteface" where you create and delete these categories. These categories are used by the _Push Notifications Mobile Client_ to "subscribe" to. For example I have a sports news site and I push messages about different sports to the mobile users. The users _subscribes_ to different sports they want to receive notifications about.
+
+With this generated project all that is left todo is connect the Unified Push Server to this project, first go to openshift.com and create an UPS instance, search on AeroGear pick a name and "Create Application". For more information on how to set up a unified push server on openshift have a look at our [documentation](/docs/unifiedpush/ups_userguide/index/#openshift). Now you have a running UPS on openshift you need to login and create an "application" that will be used by your Feedhenry apps to send push notifications. Note down the `Application ID`, `Master Secret` and the `Server URL` as we going to need those later on.
```

Comment:
@stribny mind sending a PR for the typos? I think that is way easier for all of us, not just @edewit